packages/cardano-services/src/Handle/HandleHttpService.ts

```diff
@@ -0,0 +1,24 @@
+import { HandleProvider } from '@cardano-sdk/core';
+import { HttpService } from '../Http';
+import { Logger } from 'ts-log';
+import { Router } from 'express';
+import { ServiceNames } from '../Program/programs/types';
+import { providerHandler, useOpenApi } from '../util';
+
+export interface HandleServiceDependencies {
+  handleProvider: HandleProvider;
+  logger: Logger;
+}
+
+export class HandleHttpService extends HttpService {
+  constructor({ logger, handleProvider }: HandleServiceDependencies, router: Router = Router()) {
+    super(ServiceNames.Handle, handleProvider, router, logger);
+
+    useOpenApi(__dirname, router);
+
+    router.post(
+      '/resolve',
```

Comment:
In case we have more routes in the future, and maybe for consistency, should we clean up the routes? 

`router.post(`${routes.resolve}, ...`)`

Reply:
This required some more refactoring...

anyway 4532e13d1 should solve this